environment-cluster.yml

```diff
@@ -20,11 +20,11 @@ dependencies:
         - numba>=0.56.2,<0.57
         - numexpr==2.8.4
         - kiwisolver==1.4
-        - tensorflow>=2.14,<2.15
+        - tensorflow>=2.14,<2.16
```

Comment:
I think this is not working on the cluster, since it will install version 2.15.X. This have to be <2.15.